src/lsm/compaction.zig

```diff
@@ -714,25 +715,19 @@ pub fn CompactionType(
             assert(!bar.move_table);
 
             assert(bar.value_count_per_beat == 0);
+            assert(bar.beats_max == null);
 
-            // TODO: Move this calculation into beat_grid_reserve, and subtract the values we've
-            // already done from it.
-            // This way we self correct our pacing and better spread the work out....
-            bar.value_count_per_beat = stdx.div_ceil(bar.compaction_tables_value_count, beats_max);
-            assert(bar.value_count_per_beat > 0);
-
```

Comment:
In my head, `beats_max` was always the budget, and `value_count_per_beat` was just a derivation from that. That said, I'm not particularly tied to the function name - especially since it does other stuff too.